src/externalized/DefaultContentManager.cc

```diff
@@ -1068,26 +1071,39 @@ bool DefaultContentManager::loadStrategicLayerData() {
 			BloodCatSpawnsModel::deserialize(obj)
 		);
 	}
+	delete json;
+
+	json = readJsonDataFile("strategic-map-sam-sites.json");
+	for (auto& element : json->GetArray())
+	{
+		auto samSite = SamSiteModel::deserialize(element);
+		m_samSites.push_back(samSite);
+	}
+	SamSiteModel::validateData(m_samSites);
+	delete json;
 
 	json = readJsonDataFile("strategic-map-towns.json");
 	for (auto& element : json->GetArray()) 
 	{
 		auto town = TownModel::deserialize(element);
 		m_towns.insert(std::make_pair(town->townId, town));
 	}
-	
+	delete json;
+
 	loadStringRes("strings/strategic-map-town-names", m_townNames);
 	loadStringRes("strings/strategic-map-town-name-locatives", m_townNameLocatives);
 
 	json = readJsonDataFile("strategic-map-movement-costs.json");
 	m_movementCosts = MovementCostsModel::deserialize(*json);
+	delete json;
 
 	json = readJsonDataFile("strategic-map-npc-placements.json");
 	for (auto& element : json->GetArray())
 	{
 		auto placement = NpcPlacementModel::deserialize(element);
 		m_npcPlacements.insert(std::make_pair(placement->profileId, placement));
 	}
+	delete json;
```

Comment:
most of these deletes don't belong in this PR; they're separate fixes.

Reply:
I will rewrite the commits and force-push, with the review comments and without these.